modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb

```diff
@@ -28,20 +28,25 @@ def perform(uuid, appointment_date)
 
       logger.info("Submitting travel claim for #{uuid}, #{appointment_date}")
 
-      claims_resp = TravelClaim::Service.build(
-        check_in: check_in_session,
-        params: { appointment_date: }
-      ).submit_claim
-
-      claim_number = claims_resp.dig(:data, :claimNumber)&.last(4)
+      claims_resp =
+        begin
+          TravelClaim::Service.build(
+            check_in: check_in_session,
+            params: { appointment_date: }
+          ).submit_claim
+        rescue
+          logger.error("Error calling BTSSS Service for: #{uuid}, #{appointment_date}")
+          nil
+        end
+      claim_number = claims_resp&.dig(:data, :claimNumber)&.last(4)
```

Comment:
This looks similar to what is being done within `handle_response` so maybe the same method could handle both value.

`claim_number, template_id = handle_response(claims_resp:)`